internal/updater/providers/slickvpn/location.go

```diff
@@ -0,0 +1,117 @@
+package slickvpn
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"net/http"
+	"regexp"
+
+	"github.com/qdm12/gluetun/internal/models"
+	htmlutils "github.com/qdm12/gluetun/internal/updater/utils/html"
+	"golang.org/x/net/html"
+)
+
+const (
+	ColContinent = 1
+	ColCountry   = 2
+	ColCity      = 3
+	ColConfig    = 4
+)
+
+var (
+	ErrExtractServerFromLink   = errors.New("unable to extract server string from link")
+	ErrNoMatchingServer        = errors.New("couldn't find matching server")
+	ErrOpenvpnLinkNodeNotFound = errors.New("couldn't find OpenVPN link node")
+)
+
+func fetchLocationInfo(
+	ctx context.Context,
+	client *http.Client,
+	indexURL string,
+) (serversOut []models.Server, err error) {
+	request, err := http.NewRequestWithContext(ctx, http.MethodGet, indexURL, nil)
+	if err != nil {
+		return nil, fmt.Errorf("cannot create HTTP request: %w", err)
+	}
+
+	response, err := client.Do(request)
+	if err != nil {
+		return nil, fmt.Errorf("HTTP request failed: %w", err)
+	}
+	defer response.Body.Close()
+
+	doc, err := html.Parse(response.Body)
+	if err != nil {
+		return nil, fmt.Errorf("an error occurred parsing response body: %w", err)
+	}
+
+	rows := htmlutils.GetNodesBySelector(doc, "table#location-table > tbody > tr")
+
+	rowCount := 0
+	for _, rowNode := range rows {
+		rowCount++
+		colCount := 0
+		server := models.Server{}
+		for cellNode := rowNode.FirstChild; cellNode != nil; cellNode = cellNode.NextSibling {
+			colCount++
+			switch colCount {
+			case ColCountry:
+				// Country
+				server.Country = cellNode.FirstChild.Data
+			case ColCity:
+				// City
+				server.City = cellNode.FirstChild.Data
+			case ColConfig:
+				// OVPN configs
+				linkNodes := htmlutils.GetNodesBySelector(cellNode, "a[href$='.ovpn']")
+				for _, linkNode := range linkNodes {
+					if htmlutils.GetText(linkNode) == ovpnLinkText {
```

Comment:
nit 
```suggestion
					if htmlutils.GetText(linkNode) == "OpenVPN" {
```

by the way this makes me think, do they support other protocols such as wireguard?